sigstore/_verify/verifier.py

```diff
@@ -227,7 +240,7 @@ def verify(
             entry = materials.rekor_entry(self._rekor)
         except RekorEntryMissingError:
             return RekorEntryMissing(
-                signature=materials.signature,
+                signature=base64.b64encode(materials.signature).decode(),
```

Comment:
just checking: is this change supposed to be here?

Reply:
Yep -- `RekorEntryMissing.signature` is meant to be a `str`, but we were populating it with the raw `bytes` from the materials by accident. I noticed that while (accidentally) trigging the error case by mixing up different Sigstore instances 🙂